apps/server/migrations/0202_add_mentions_to_comments.sql

```diff
@@ -0,0 +1,2 @@
+-- Add mentions field to comments table with a length constraint
+ALTER TABLE "comments" ADD COLUMN IF NOT EXISTS "mentions" varchar(500); 
```

Comment:
you'd probably want a varchar[] type field here which is hard mode

Reply:
actually the complexity here is you can change your username so really what you'd want is something more annoying like mentioned user ids but you kind of need a mention username to id mapping

could just replace the mentions in the comments with some special syntax which is the ID, then replace them on output as well? not totally sure